build/configs/artik053/tools/openocd/partition_map_romfs.cfg

```diff
@@ -0,0 +1,22 @@
+#
+# Partition Map for ROMFS file system
+#
+# WARNING:
+# The size of each partition must be same as CONFIG_ARTIK053_FLASH_PART_LIST config in menuconfig.
+# In this file, the size of the user partition is set to 1000KB and that of the rom partition is set to 400KB.
+# If you change CONFIG_ARTIK053_FLASH_PART_LIST config in menuconfig, you must modify this file.
+#	Name	  Description		Start address	Size		RO
+set partition_list {
+	bl1	{ "BL1"			0x04000000	0x00004000	0 }
+	sssro	{ "SSS R/O Key"		0x04004000	0x0000C000	1 }
+	bl2	{ "BL2"			0x04010000	0x00030000	0 }
+	sssfw	{ "SSS F/W"		0x04040000	0x00008000	0 }
+	wlanfw	{ "WLAN F/W"		0x04048000	0x00080000	0 }
+	os	{ "OS"			0x040C8000	0x00258000	0 }
+	factory	{ "Factory Reset"	0x04320000	0x00180000	0 }
+	ota	{ "OTA download"	0x044A0000	0x00180000	0 }
+	user 	{ "USER R/W" 		0x04620000 	0x000FA000  	0 }
+	rom  	{ "ROM FS"  		0x0471A000 	0x00064000  	0 }
```

Comment:
Is there a way to address potential mismatches between partition layout here and in a defconfig file?

Reply:
I second the comment of  tadziopazur. We may need to have a method to find the mismatches of sizes defined here and in defconfig.

Reply:
I agree. But this commit branch is just for adding a ROMFS partition. The way to address potential mismatches is needed, but not on here.